memory/toml_tree_store.py

```
@@ -0,0 +1,816 @@
+            d["source"] = self.source
+        return d
+
+    def to_full_dict(self) -> dict:
+        """序列化为完整格式（含运行时 meta，用于归档/API）"""
+        d = self.to_toml_dict()
+        d["meta"] = self.meta
+        return d
+
+    @classmethod
+    def from_toml_dict(
+        cls, data: dict, runtime_meta: dict = None, **location_kwargs
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)